Eastern-Research-Group/mywaterway
# Feature/162 add monitoring columns

## Related Issues:
* https://jira.epa.gov/browse/HMW-162

## Main Changes:
* Renamed the `Count` column to `Location Count`
* Added the `Sample Count` and `Measurement Count` columns.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/monitoring
2. Click on the Sample Locations tab
3. Verify the table has the `Location Count`, `Sample Count`, and `Measurement Count` columns.

## TODO:
- [x] PR #601 needs to be merged in before this one can be merged.
